vault/resource_quota_rate_limit.go

```diff
@@ -112,6 +133,14 @@ func quotaRateLimitUpdate(d *schema.ResourceData, meta interface{}) error {
 	data["path"] = d.Get("path").(string)
 	data["rate"] = d.Get("rate").(float64)
 
+	if intervalRaw, ok := d.GetOk("interval"); ok {
+		data["interval"] = intervalRaw.(int)
+	}
+
+	if blockIntervalRaw, ok := d.GetOk("block_interval"); ok {
+		data["block_interval"] = blockIntervalRaw.(int)
+	}
```

Comment:
Same suggestion for dropping the type assert above

```suggestion
	if v, ok := d.GetOk("interval"); ok {
		data["interval"] = v
	}

	if v, ok := d.GetOk("block_interval"); ok {
		data["block_interval"] = v
	}
```